plotly/grid_objs/grid_objs.py

```diff
@@ -187,3 +254,17 @@ def get_column(self, column_name):
         for column in self._columns:
             if column.name == column_name:
                 return column
+
+    def get_fid_uid(self, column_name):
```

Comment:
Why is it called `get_fid_uid`? Maybe just `get_uid`? Also, are users supposed to use this? They shouldn't need to know about `uid`s or `src`s. If it's something users are playing with, I'd suggest `get_column_reference` or something (which should return the `fid:uid` pair (and could potentially handle `2d` references).